mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -311,24 +311,26 @@ bool GlAnimationOverlayCalculator::LoadAnimationAndroid(
     }
 
     // Set triangle_mesh normals for shader usage
-    int len = lengths[2];
-    triangle_mesh.normals.reset(new float[len]);
-    float triangle_normals[len];
+    triangle_mesh.normals.reset(new float[lengths[0]]);
 
-    // Compute every triangle surface normal and store them for later use.
-    for (int idx = 0; idx < len; idx += 3) {
+    // Used for storing the vertex normals prior to averaging
+    float vertex_normals_sum[lengths[0]];
+    float vertex_avg_denom[lengths[0]];
```

Comment:
nit: might as well have the denom just be an integer-- or can even call it "*_count" to signify we're just counting up